cvmfs-config-osg.service

```diff
@@ -0,0 +1,11 @@
+[Unit]
+Description=daemon to keep /cvmfs/config-osg.opensciencegrid.org mounted
+After=autofs.service
+
+[Service]
+Type=simple
+ExecStart=/usr/sbin/cvmfs-config-osgd -p /run/cvmfs-config-osgd.pid /cvmfs/config-osg.opensciencegrid.org
+PIDFile=/run/cvmfs-config-osgd.pid
```

Comment:
* I think this should have a `Requires` statement for `autofs`.  `After` only designates only the sequencing, not the dependency.
* Might also be possible to put a requirement on the `/cvmfs` mount itself.
* Automatic restart? 
* Is the PID file actually necessary for a service of type `simple`?